### PR TITLE
Problem reporting website: opts must include a valid :workspace. 

### PR DESCRIPTION
**Pull request by exenin**
https://github.com/rapid7/metasploit-framework/issues/10999

### CODE REVIEW

File: plugins/wmap.rb [hunk 1]
```diff
@@ -1416,7 +1416,7 @@ def process_urls(urlstr)
         end
 
         #print_status "aaa"
-        #print_status framework.db.workspace.name
+        print_status framework.db.workspace.name
```
Comment (bcoles) [inline plugins/wmap.rb]: Was this uncommenting intentional?

**Comment by bcoles** [comment]
It is required that code in your fork be merged **from a unique branch** in your repository to master in Rapid7's.  Please create a new branch in your fork of framework and resubmit this from that branch.

```
git checkout -b <BRANCH_NAME>
git push <your_fork_remote> <BRANCH_NAME>
```

This helps protect the process, ensure users are aware of commits on the branch being considered for merge, allows for a location for more commits to be offered without mingling with other contributor changes and allows contributors to make progress while a PR is still being reviewed.

Closing based on the this requirement, please do resubmit from a unique branch.

File: plugins/wmap.rb (hunk 1 above)
Comment (threedr3am) [reply plugins/wmap.rb]: Excuseme,the problem how to slove?

Comment (threedr3am) [reply plugins/wmap.rb]: I pull head master now.But,this problem not slove.

**Comment by bcoles** [comment]
@exenin are you keen to submit a new PR? Alternatively, we can take care of patching the issue. I've verified your patch works as described.

**Comment by agaryen** [comment]
@bcoles: Has this been fixed yet?

**Comment by bcoles** [comment]
@agaryen it has not been fixed. I scared off @exenin

**Comment by bcoles** [comment]
#11284

**Pull request closed without merging**